Services/UserService/Utils/EmailSender.cs

```diff
@@ -33,19 +34,25 @@ public void SendEmail(string email, ISecretTokenEngine secretTokenEngine)
 
             var flag=true;
 
-            for (int i = 0, k = 1; i < k; ++i)
+            for (int i = 0; i < 5; ++i)
             {
                 try
                 {
                     flag = true;
                     smtp.Send(m);
+                    break;
                 }
-                catch (SmtpException)
+                catch (SmtpException e)
                 {
-                    Thread.Sleep(10000 * k);
+                    Thread.Sleep(10000 * (++i));
```

Comment:
why not to use `Thread.Sleep(10000)`

Reply:
Because we wont to have more time after every try